x-pack/plugins/osquery/server/routes/action/create_action_route.ts

```diff
@@ -19,17 +19,34 @@ export const createActionRoute = (router: IRouter, osqueryContext: OsqueryAppCon
     {
       path: '/internal/osquery/action',
       validate: {
-        params: schema.object({}, { unknowns: 'allow' }),
-        body: schema.object({}, { unknowns: 'allow' }),
+        body: schema.object({
```

Comment:
maybe we could move it to a common folder, so then we can use it to validate both, ui and server side?
https://github.com/elastic/kibana/blob/master/x-pack/plugins/osquery/common/schemas/routes/saved_query/create_saved_query_request_schema.ts